edx_adapt/tests/performance_tests.py

```diff
@@ -0,0 +1,86 @@
+import unittest
```

Comment:
Actually we don't need this file at all. Please delete it.

Performance testing is done in other place. See `taurus_load_tests.yml` in master branch.